File: api/code-samples/python/klay/filter/getFilterChanges.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x63000c8f11e790cb4c93b0659319a159"

sdk = OpenSDK(host)
klay_response = sdk.klay.get_filter_changes(quantity)

print(klay_response)

File: web3rpc/rpc-specs/api/code-samples/javascript/klay/filter/getFilterChanges.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const quantity = "0x16"
    sdk.klay.getFilterChanges(quantity, {}, (err, data, response) => {
        console.log(data);
    });
}
)()